<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_folder_scan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Repertoire a scaner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_folder_destination">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_folder_scan">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose The Source Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_folder_destination">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose The Target label Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="champ_minimal_file_size">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tailml_minimal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Taill min De fichier </string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_scann">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Scann And Copie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_annuler">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_folder_scan_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Repertoire a scaner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tailml_minimal_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>41</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Ko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_msg_err_size">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Aucun Téléchargement En Cours</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
